tests/xtimer_msg_receive_timeout/main.c

```diff
@@ -0,0 +1,24 @@
+#include "thread.h"
+#include "msg.h"
+#include "xtimer.h"
+#include "timex.h"
+
+int main(void)
+{
+    msg_t m, tmsg;
+    xtimer_t t;
+    int64_t offset = -1000;
+    tmsg.type = 44;
+
+    while (1) {
+        xtimer_set_msg64(&t, SEC_IN_USEC + offset, &tmsg, sched_active_pid);
+        if (xtimer_msg_receive_timeout64(&m, SEC_IN_USEC) < 0) {
```

Comment:
Hm, I thought about, but couldn't find a nice solution. Actually, it would be probably more sensible to use the 32-bit version here. But for really testing the 64-bit version, one would have to wait quite a while...